tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -1288,7 +1282,7 @@ public function testCreateContributionPendingOnline() {
     $this->assertEquals($contribution['values'][$contribution['id']]['trxn_id'], 12345);
     $this->assertEquals($contribution['values'][$contribution['id']]['invoice_id'], 67890);
     $this->assertEquals($contribution['values'][$contribution['id']]['source'], 'SSF');
-    $this->assertEquals($contribution['values'][$contribution['id']]['contribution_status_id'], 2);
+    $this->assertEquals(2, $contribution['values'][$contribution['id']]['contribution_status_id']);
```

Comment:
OK - I realise all the lines  above need the same change - I decided after the first one not to get carried away